crud/src/test/java/com/redhat/lightblue/mediator/MediatorTest.java

```diff
@@ -401,6 +400,72 @@ public void queryTimeoutTest() throws Exception {
     }
 
     @Test
+    public void findResultSetTooLargeTest() throws Exception {
```

Comment:
> Can we add a test for update/insert/delete when there are no hooks registered?
There is one already: MediatorTest#updateResultSetTooLargeTest. No save and delete, but they use the same code path for this.

> Might even want to test even with hooks, to ensure the hookManager result size checks are setup correctly.

I agree, the components/units are tested in isolation, but their integration is not tested. On the other hand the setup logic is really simple.